client/components/mma/accountoverview/ProductCard.tsx

```diff
@@ -204,6 +204,15 @@ export const ProductCard = ({
 		productDetail.subscription.nextPaymentDate !==
 			productDetail.subscription.potentialCancellationDate;
 
+	const futureProductTitle =
+		productDetail.subscription.futurePlans.length > 0 &&
+		productDetail.subscription.futurePlans[0].tier &&
```

Comment:
nit pick-just wondering if these two lines could be combined with some kind of question mark, I'm not sure the exact syntax but this kind of thing
```suggestion
		productDetail.subscription.futurePlans[0]?.tier &&
```